libs/dh/globalization/assets-localization/src/index.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import da from './assets/i18n/da.json';
+import en from './assets/i18n/en.json';
+
+export { da, en };
```

Comment:
suggestion: we might be able to refactor to something like:
```suggestion
export da from './assets/i18n/da.json';
export en from './assets/i18n/en.json';
```

Reply:
Actually that will make typescript complaining, also `export * as da ...` won't work either :/

Reply:
Probably because JSON imports are special. Let's stick with the current solution.